dir/path.go

```diff
@@ -79,21 +79,35 @@ const (
 
 var userConfigDir = os.UserConfigDir // for unit test
 
-func init() {
-	loadUserPath()
-}
-
 // loadUserPath function defines UserConfigDir and UserLibexecDir.
 func loadUserPath() {
```

Comment:
Looks like we don't need this function now. If we don't need it, we can remove it as it is an internal function.

Reply:
Resolved in [46c35d4](https://github.com/notaryproject/notation-go/pull/449/commits/46c35d45a2c830ad16861f7ab00b6a9ccb8aec9d).